### ISSUE TITLE
When will the training code be available?

### ISSUE THREAD

**Issue by Xiangyu-CAS** [umich-vl/pose-ae-demo#1]
Hi,  Mr Deng,  do you plan to release the complete training code in the future ? A implementation in tensorflow is popular for novice like me。

**Comment by jin-miao-cai** [comment]
Hi. Mr Deng, I have try to reproduce the work in human pose estimation, but I get some problem about the loss function for embedding map. The value of the generated embedding map is so small and there is almost no absolute value distance between the reference embedding of different person. I have try a lot to deal with this problem, but it does not work. 
So, do you plan to release more detail about the loss function？I will be very grateful about that.

**Comment by jin-miao-cai** [comment]
Hi. Mr Deng, I have try to reproduce the work in human pose estimation, but I get some problem about the loss function for embedding map. The value of the generated embedding map is so small and there is almost no absolute value distance between the reference embedding of different person. I have try a lot to deal with this problem, but it does not work.
So, do you plan to release more detail about the loss function？I will be very grateful about that.

**Comment by wangxuanhan** [comment]
@jin-miao-cai Hi, i met the same issue before. I changed the penalty for group loss where the first penalty should be normalized by a smaller weight(i.e., 1/(N*K) instead of 1/N). It seems work, but it will hurt the precision on heatmap regression.

**Comment by anewell** [comment]
The training code is now up: https://github.com/umich-vl/pose-ae-train

**Comment by Xiangyu-CAS** [comment]
@anewell Great!  Thanks so much~

**Comment by jin-miao-cai** [comment]
@anewell thanks a lot !
